### ISSUE TITLE
Output "Just now" instead of "0 seconds ago" in strict mode

### ISSUE THREAD

**Issue by Sans84** [marcreichel/alpine-timeago#11]
Is it possible to display "Just now" instead of "0 seconds ago" for a newly added record in strict mode?

**Comment by marcreichel** [comment]
@Sans84 I don't think this is possible. This package depends on `date-fns` to display the text. The only way of implementing this here directly would be to catch that case in this package before letting `date-fns` do its thing. The problem here would be that I would have to maintain the “Just now” string for every language, which would bloat up the size of this package.

In my opinion the better way would be to create an issue (or PR) in the `date-fns` repository, for them to implement this feature.

The `alpine-timeago` package can then adapt synchronously.
